tests/pipelines/main.from_assemblies.nf.test

```diff
@@ -21,14 +22,16 @@ nextflow_pipeline {
 
                 skip_bakta = true
                 skip_mobrecon = true
-                skip_checkm = true
+                skip_checkm = false
```

Comment:
I just noticed that in many places you've changed checkm variables to checkm2, but the flag here is still checkm. I'm not sure if that's intentional or not.